src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -293,6 +296,11 @@ protected Task OnAckMessageAsync(AckMessage ackMessage)
             return Task.CompletedTask;
         }
 
+        private Task OnEventMessageAsync(ServiceEventMessage message)
+        {
+            return _serviceEventHandler?.HandleAsync(ConnectionId, message) ?? Task.CompletedTask;
```

Comment:
shall we make it `_ =` or `Task.Run` so that even if customer implemented his own event handler, it does not block us.